### ISSUE TITLE
Archived Downloads is broken

### ISSUE THREAD

**Issue by sklm** [BiglySoftware/BiglyBT#572]
Kubuntu 18.04 LTS, KDE
BiglyBT 1.6.0.0 and up to and including beta 03.

Archived Downloads is broken to the point where it's even hard to explain what's wrong with it. For starters, there are multiple entries being created under Archived Downloads for each archived torrent; I suspect it's one entry per file in the torrent. Unfortunately, I just started archiving of 800 torrents and BiglyBT is now in the middle of adding thousands and thousands of entries under Archived Downloads. On top of that, UI is blank and not refreshing so I can't see those entries but I see the counter incrementing.

**Comment by parg** [comment]
Nice :) The issue doesn't occur with the 'fancy' context menu but it does with the classic one - interesting! Fix will be in the next beta

**Comment by sklm** [comment]
Indeed, with fancy menu the archiving works fine. Thank you in advance for fixing it.

**Comment by sklm** [comment]
I also noticed that even when the Fancy Menu is globally enabled, it still looks like the classic one when invoked on torrents in Archived Downloads pane.

**Comment by parg** [comment]
Yup, currently the fancy one only applies to the various 'real' torrent views - the non-fancy one is also available via the main 'Torrent' menu (if you enable it in settings) and also if you double-right-click in the torrent views.
